src/pages/Home/hooks/useScrollEvent.tsx

```diff
@@ -1,17 +1,24 @@
-import { useEffect } from 'react';
+import { useEffect, useRef } from 'react';
 
 type ScrollHandler = () => void;
 
 const useScrollEvent = (onScroll: ScrollHandler) => {
+  const rafRef = useRef<number>();
+
   useEffect(() => {
     const handleScroll = (event: Event) => {
-      onScroll();
+      rafRef.current = requestAnimationFrame(() => {
+        onScroll();
+      });
```

Comment:
사소한데 이렇게 바꿔볼 수 있을 것 같아요 ㅎㅎ
```suggestion
      rafRef.current = requestAnimationFrame(onScroll);
```

Reply:
고마워요 디테일한 피드백 :)